### DISCUSSION TITLE
Making the UI more modern

### DISCUSSION THREAD

**Discussion by guillaumechereau** [guillaumechereau/goxel#333] [Ideas]
I am thinking of redoing the UI a bit, to make it look more modern, with flat icons.

Inspired by Blender, I made this mokup to get an idea of what I would like to get.  Feel free to comment.

![goxel-test-design](https://github.com/guillaumechereau/goxel/assets/107679/07ef1fe4-d224-423f-8e3b-eb9bdc95a8a1)

**Comment by JasperMoerland** [comment]
this looks really cool!

**Comment by guillaumechereau** [comment]
I already pushed in master the current branch, so the next release will have the updated UI.  Here is what it looks like at the moment (dark and light modes:)

<img width="896" alt="Screenshot 2024-01-07 at 1 19 16 AM" src="https://github.com/guillaumechereau/goxel/assets/107679/eea7620b-a9c6-4069-9c5d-728b0212d835">

<img width="898" alt="Screenshot 2024-01-07 at 1 19 59 AM" src="https://github.com/guillaumechereau/goxel/assets/107679/d5549a08-39ad-43e4-b83c-d40355af573a">

One features I added is the possibility to detach the panels so that we can move them around and have more than one opened at the same time.

There are still a few issues to fix before can make a release though.

**Reply by pegvin** [reply]
Looks really good but the "Accent Blue" color should've been a bit more brighter in the Light Theme, to make the Black Text color pop more.

**Reply by guillaumechereau** [reply]
Right, I'll try to adjust it.  I am not really good with colors.

**Reply by pegvin** [reply]
Nor am I, Thanks for all the efforts you put into open-source.

**Comment by YarlBoro** [comment]
Somehow I've missed this post.  I've just checked latest release with new UI. It is giant step up, fantastic!
